### PR TITLE
Updates to About Page 

### PR DESCRIPTION
**Pull request by Akhan221**
## Description

Updated the about pages by splitting it into two new pages on the navbar, aboutGTHC and aboutus. AboutGTHC contains an origin story and the purpose of GTHC. Aboutus introduces the team with profile cards listing graduation class and team role. Extraneous information on the aboutus page was removed as well, such as the appreciation post for Debbie and K-Ville logos.

### CODE REVIEW

File: app/javascript/src/containers/About.jsx [hunk 1]
```diff
@@ -25,119 +26,45 @@ class About extends Component {
...
+<br></br><br></br>
+In Fall of 2018, Vinit Parekh, the founder of Fix My Campus at Duke, was given the opportunity to propose the idea of a Kville App to Duke Basketball with the help of Professor Duvall's Computer Science 408 course, which connected students to real-world clients to build software solutions. Vinit came to the class as a client and proposed to the group of students; among the group of students sat Aman, who that day had begun moving on from the project as no progress was made in months, and thought 408 would be a great way to find an excuse to build software. As if it was fate, through the 408 class, Vinit, Aman, Anesu, and Rikki Kendall began working together to build the first iteration of GTHC for the 2019 tenting season.</h2>
+<br></br><br></br>
```
Comment (amanmibra) [inline app/javascript/src/containers/About.jsx]: Fix the alignment of these lines. Tab them into place.

File: app/javascript/src/containers/About.jsx [hunk 2]
```diff
@@ -25,119 +26,45 @@ class About extends Component {
...
-                  <Card.Header>Aman Ibrahim</Card.Header>
-                  <Card.Meta>One of the Three Amigos</Card.Meta>
-                  <Card.Description> Aman Ibrahim, a North Carolina native, is a Duke student in the class of 2020. He is a Computer Science major, and is working towards an Islamic Studies Certificate and an Electrical and Computer Engineering minor. He is incredibly versed in web development and is a full stack engineer. </Card.Description>
+              <Card.Content textAlign='left'>
+                <h2>The story of GTHC begins in February 2018 when Aman Ibrahim, then a Duke sophomore, experienced his first tenting season awaiting the Duke vs. UNC men's basketball game. Two weeks before the end of tenting, his Dirty Black tent was booted from Krzyzewskiville for missing two nightly tent checks. Aman felt this was due to the team's not-so-organized method of scheduling tent shifts over Google Sheets. Aman was surprised he couldn't find an alternative solution, so after being inspired by another peer, he decided it would be a cool, fun, but not serious side project building out a platform that would ease the difficulties of managing a K-Ville tent.
```
Comment (amanmibra) [inline app/javascript/src/containers/About.jsx]: Use a `p` (paragraph) element instead of `h2`

File: app/javascript/src/containers/About.jsx [hunk 3]
```diff
@@ -25,119 +26,45 @@ class About extends Component {
...
+          </Card>
+          </Container>
+          </div>
+        
+          
+          <div className="mission" padding-top="48px">
+          <Container textalign="center" >
+          <Header 
+            size='huge'
+            content="What is GTHC?"
```
Comment (amanmibra) [inline app/javascript/src/containers/About.jsx]: Switch "What is GTHC?" and "Origins of GTHC"

File: app/javascript/src/containers/AboutUs.jsx [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+                <Card centered>
+                  <Image rounded src='https://i.imgur.com/BJNB5pt.jpg'/>
+                  <Card.Content>
+                    <Card.Header> Shamikh Hossain </Card.Header>
+                    <Card.Meta>Class of 2020</Card.Meta>
+                  </Card.Content>
+                  <Card.Content extra>
+                    <a>
+                      <Icon name='user' />
+                      Team Member
```
Comment (amanmibra) [inline app/javascript/src/containers/AboutUs.jsx]: Change "Team Member" and "Tern" to either "Developer", "Marketing", or "Junior Developer"

File: config/routes.rb [hunk 5]
```diff
@@ -11,6 +11,11 @@
   get 'app/*path', to: 'pages#index'
   get 'tenting101', to: 'pages#index'
   get 'about', to: 'pages#index'
+  get 'about/aboutgthc', to: 'pages#index'
+  get 'about/aboutus', to: 'pages#index'
+  # get 'about/*path', to: 'pages#index'
+  # get 'about/aboutus', to: 'about#aboutus'
```
Comment (amanmibra) [inline config/routes.rb]: remove these comments

**Review by amanmibra — CHANGES_REQUESTED** [review]
Good work, just make these small changes, and ping me when you finished them. :+1:

**Comment by Akhan221** [comment]
Made changes to the about pages as requested, however the console keeps giving the following warning for the image alignment despite troubleshooting: 

Received `true` for a non-boolean attribute `centered`

**Review by amanmibra — APPROVED** [review]
Good Work! Congrats on your first official GTHC PR! 👍 💯 🥇

**Pull request merged**